src/main/java/net/snowflake/client/config/SFClientConfigParser.java

```diff
@@ -58,6 +64,12 @@ public static SFClientConfig loadSFClientConfig(String configFilePath) throws IO
         File configFile = new File(derivedConfigFilePath);
         ObjectMapper objectMapper = new ObjectMapper();
         SFClientConfig clientConfig = objectMapper.readValue(configFile, SFClientConfig.class);
+        Set<String> unknownParams = clientConfig.getUnknownParamKeys();
+        if (!unknownParams.isEmpty()) {
+          for (String unknownParam : unknownParams) {
+            logger.warn("Unknown field from config: {}", unknownParam);
```

Comment:
could we change it e.g. to `Unknown configuration option from configuration file: {}`?

Reply:
In the other drivers we use the message in such format. This would keep the parity between them.